static/_redirects

```diff
@@ -56,3 +56,6 @@
 
 /docs/open-source-quick-start/quick-start-installation-guide-on-gcp/summary-of-what-you-have-deployed-gcp/ /docs/open-source-quick-start/quick-start-installation-guide-on-gcp/summary-of-what-you-have-deployed/ 301
 /docs/getting-started-on-snowplow-open-source/quick-start-gcp/query-your-postgres-data/ /docs/getting-started-on-snowplow-open-source/quick-start-gcp/query-your-data/ 301
+
+/docs/getting-started-on-snowplow-open-source/setup-snowplow-on-gcp/setup-google-cloud-storage-gcs-destination/downloading-google-cloud-storage-loader/ /getting-started-on-snowplow-open-source/setup-snowplow-on-gcp/setup-destinations/google-cloud-storage/ 301
```

Comment:
Something is off here:
* Does `setup-destinations/google-cloud-storage/` exist in this PR?
* The redirect location should start with `/docs`

Reply:
Sorry about that. I've updated and tested them now.